### DISCUSSION TITLE
java-features/java21/emoji-support

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#8] [Announcements]
# java-features/java21/emoji-support

Java 21增强对Emoji表情符号的处理了 发现一个 Java 21 中有意思的东西！在java.Lang.Character类中增加了用于确定字符是否为 Emoji 表情符号的 API，主要包含下面六个新的静态方法： public static boolean isEmoji(int codePoint) { return CharacterData.of(codePoint).isEmoji(codePoint); } public static boolean isEmojiPresentation(int codePoint) { return CharacterData.of(codePoint).isEmojiPresentation(codePoint); } public static boolean isEmojiModifier(int codePoint) { return CharacterData.of(codePoint).isEmojiModifier(codePoint); } public static boolean isEmojiModifierBase(int codePoint) { return CharacterData.of(codePoint).isEmojiModifierBase(codePoint); } public static boolean isEmojiComponent(int codePoint) { return CharacterData.of(codePoint).isEmojiComponent(codePoint); } public static boolean isExtendedPictographic(int codePoint) { return CharacterData.of(codePoint).isExtendedPictographic(codePoint); }

https://www.didispace.com/java-features/java21/emoji-support.html

**Comment by toutzhang** [comment]
左边的目录『增强对moji的支持』不对哦
